_posts/2020-11-08-the-sunk-cost-fallacy.md

```diff
@@ -0,0 +1,110 @@
+
+"Why not rewrite the implementation code of this feature with a much simpler and leaner design ?."
+
+But another thought hit me which threw me into a dilemma.
+
+“We invested so much time working on this for a couple of months. Even though the design is complicated, we should continue following it since we are already midway. It will anyway fulfil the requirement even if it takes longer. “
+
+I figured that my team would come together to figure out a solution for this dilemma. I liked the opinion of one of the team members.
+
+“The initial complicated design consisted of so many assumptions out of which some were revealed to be not true. We had to anyway rethink those parts of design again. This initial design also aimed to be inclusive of future milestones which we were never certain that they would get approved by the management. Let's rewrite this feature code with narrowed down to cover just the customer requirements and not think much of the non-deterministic future and milestones.”
+
+We chose to rewrite this feature code with a simple design. We were on the brink of burning up but we still had a ray of hope that we can revive this project.
+
+“We got this, man!”, one of them said.
+
+“We got this!”.
+
+## T + 7 - The end
+
+It was a pleasant morning. I had 10 minutes left for the release meeting. A quote from my favourite game, Skyrim struck me.
+
+_"I fight so that all the fighting I've already done hasn't been for nothing. I fight... because I must."_ 
+
+The king wanted to keep fighting even after he lost half of his men, just so that their sacrifices don't go to waste.
+
+Economists coined a term for this, **“sunk cost fallacy”**.  
+
+The Decision Lab defined it as
+
+**“The Sunk Cost Fallacy describes our tendency to follow through on an endeavor if we have already invested time, effort or money into it, whether or not the current costs outweigh the benefits.”**
+
+There are various ways to avoid this, the easiest being to let go of your fear and attachment to failing projects and make room for better brighter things. And who knows, this might actually pave out a way to further successful and profitable projects. 
+ 
+It was time for my meeting. My manager was about to announce the completion of this feature which we devoted several months on.
+
+“So finally we managed to complete this feature this release”. 
+
+We sighed with relief. We had just finished the most complicated and long-lasting feature project our team had ever faced. One of my team members even took a long holiday. We all were celebrating in our ways. 
+
+I quickly glanced through my diary. It had scribbles of the lessons I had learnt. They were written as follows:
+
+ - It is okay to let go of past costs if you think future investments are greater than the former. Don't assume things. Take note of “known unknowns” and find answers to them before making major decisions.
```

Comment:
I think, you are using costs and investments interchangeably, I can't make out the intent of the sentence